src/api/app/views/webui/user/_dropdown_menu.html.haml

```diff
@@ -16,7 +16,7 @@
     %li
       = sprite_tag('cancel')
       = link_to('Delete user', {controller: "user", action: "delete", user: user.login }, |
-                method: :delete, data: { confirm: "Are you sure?" }) |
+                method: :post, data: { confirm: "Are you sure?" }) |
```

Comment:
:thought_balloon:  why?

Reply:
Looks like I messed up a rebase. I was refactoring this part in #4631

Reply:
@hennevogel updated